pycbc/strain.py

```diff
@@ -562,6 +569,10 @@ def insert_strain_option_group_multi_ifo(parser):
                             action=MultiDetOptionAction, metavar='IFO:SEED',
                             help="Seed value for the generation of fake "
                             "colored gaussian noise")
+    data_reading_group.add_argument("--fake-strain-from-file", nargs="+",
+                            action=MultiDetOptionAction, metavar='IFO:FILE',
+                            help="File containing ASD for generating fake "
+                            "noise from it.")
```

Comment:
You get the:

![seal_of_approval](https://cloud.githubusercontent.com/assets/11438929/16523344/424798e0-3fa3-11e6-8df3-627f87191a04.jpg)

for updating the multi-ifo option group as well!